### ISSUE TITLE
复现成功率问题

### ISSUE THREAD

**Issue by zimaozhang** [MINT-SJTU/VLA-Pruner#10]
作者您好，我在复现openvla的剪枝过程中使用
CUDA_VISIBLE_DEVICES=0 python experiments/robot/libero/run_libero_eval.py \
    --use_text_vision_selection False \
    --use_prefil_attention True \
    --use_temporal True \
    --pretrained_checkpoint checkpoints/openvla-7b-finetuned-libero-spatial \
    --use_fastv True \
     --task_suite_name libero_spatial \
     --fastv_r 0.875 \
     --seed 7 \
     --run_id_note vlapruner_prefill_12.5% \
     --num_trials_per_task 50
我使用这个指令在spatial的数据集上做评估，其成功率并没有达到80%左右，而是只有69%，并且goal上也只有62%并没有达到69%，请问是可能是哪里出现了问题？

**Comment by ziyanliux** [comment]
方便提供一下实验环境信息吗？例如显卡型号，以及 Python、PyTorch、CUDA、LIBERO 等相关环境版本。

另外，是否也测试了相同剪枝率（fastv_r=0.875）下的 FastV 结果，以及未剪枝的 OpenVLA 原始结果？如果有的话，也可以提供下

以及 --use_prefil_attention True 建议关闭，很多情况下效果会更好

**Comment by zimaozhang** [comment]
您好我的实验环境是4090 cuda12.1 python3.10.16 torch 2.2.0 mujoco 3.3.4 libero 0,1.0
我的spatial未剪枝是84%，在我关闭这个--use_prefil_attention即--use_prefil_attention False效果确实变好了一点达到76.8%，但还是差点

**Comment by gw103** [comment]
> 作者您好，我在复现openvla的剪枝过程中使用 CUDA_VISIBLE_DEVICES=0 python experiments/robot/libero/run_libero_eval.py --use_text_vision_selection False --use_prefil_attention True --use_temporal True --pretrained_checkpoint checkpoints/openvla-7b-finetuned-libero-spatial --use_fastv True --task_suite_name libero_spatial --fastv_r 0.875 --seed 7 --run_id_note vlapruner_prefill_12.5% --num_trials_per_task 50 我使用这个指令在spatial的数据集上做评估，其成功率并没有达到80%左右，而是只有69%，并且goal上也只有62%并没有达到69%，请问是可能是哪里出现了问题？

我也遇到了，87.5我在spatial复现的成功率是70.8%

**Comment by ziyanliux** [comment]
同样的，use_prefil_attention True 建议关闭，不大稳定

**Comment by gw103** [comment]
> 同样的，use_prefil_attention True 建议关闭，不大稳定

好的，谢谢您

**Comment by ziyanliux** [comment]
没事没事，有问题随时交流

**Comment by zimaozhang** [comment]
您好，我关闭了，但在spatial没有达到80%的效果，是什么原因，是否是版本的原因，希望您告知我，您的cuda版本，mujuco版本，torch版本等信息



发自我的iPhone


------------------ 原始邮件 ------------------
发件人: ziyanliu ***@***.***&gt;
发送时间: 2026年6月17日 09:42
收件人: MINT-SJTU/VLA-Pruner ***@***.***&gt;
抄送: zimaozhang ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [MINT-SJTU/VLA-Pruner] 复现成功率问题 (Issue #10)



ziyanliux left a comment (MINT-SJTU/VLA-Pruner#10)
 
同样的，use_prefil_attention True 建议关闭，不大稳定
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications, keep track of coding agent tasks and review pull requests on the go with GitHub Mobile for iOS and Android. Download it today!

**Comment by ziyanliux** [comment]
可以保留一下这个issue，应该不少人会遇到这个问题，我后续确认下配置发过来。
如果有评测需求，保证环境相同评测公正前提下，使用自己平台的复现结果即可。
